titus-api/src/main/java/com/netflix/titus/api/jobmanager/model/job/ebs/EbsVolume.java

```diff
@@ -0,0 +1,184 @@
+/*
+ * Copyright 2020 Netflix, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.titus.api.jobmanager.model.job.ebs;
+
+import java.util.Objects;
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Size;
+
+import com.google.common.base.Preconditions;
+
+public class EbsVolume {
+
+    public enum MountPerm {RO, RW}
+
+    public EbsVolume(String volumeId, String volumeAvailabilityZone, int volumeCapacityGB, String mountPath, MountPerm mountPerm, String fsType) {
+        this.volumeId = volumeId;
+        this.volumeAvailabilityZone = volumeAvailabilityZone;
+        this.volumeCapacityGB = volumeCapacityGB;
```

Comment:
Should we use the same unit (MB) as the `disk` resource?

Reply:
The minimum EBS volume size is GB, so I decided to leave it as GB as that is the smallest unit possible unit. Also, this value only translates to the size specified on the EBS PersistentVolume object, so doesn't map or relate to anything in the container's diskMB.